Design changes and improvements

shirishgoyal opened the pull request:
Bootstrap UI code removal
MD based design changes
Status based color assignment to tasks

![image](https://cloud.githubusercontent.com/assets/315012/9667344/dcdac664-5298-11e5-99e9-a355858d51f7.png)

![image](https://cloud.githubusercontent.com/assets/315012/9667352/e80fb21a-5298-11e5-800f-b74b80db326b.png)

![image](https://cloud.githubusercontent.com/assets/315012/9667356/ec25b462-5298-11e5-9e78-3f8300543ec0.png)

![image](https://cloud.githubusercontent.com/assets/315012/9667379/0b82e866-5299-11e5-8b92-5ccf49cc5ee5.png)

dmorina commented:
layout is good, we should make the feed content centered, we can disable preview for now as we dont have more info to show; project card can be .75*width of that (too much white space between columns)
max number of colors if 2 to 3 (1 being primary, 2 secondary and third accent) statuses, errors are an exception; I would suggest not putting colors on table headers

shirishgoyal commented:
Layout for task-feed is centered if no task is selected.

dmorina commented:
ok, cool just disable selection then, it's not good ux if content changes position {width height change is ok}

shirishgoyal commented:
For task selection, also thinking of doing it like below

![image](https://cloud.githubusercontent.com/assets/315012/9667723/2ab763fe-529b-11e5-9039-c1bbfbfc6fa2.png)

shirishgoyal commented:
Updated theme as per agreed palette 
- #FBF5AB (yellow)
- #DEE4EE (grey)
- #243058 (dark blue).

![image](https://cloud.githubusercontent.com/assets/315012/9699261/dba1a800-53f9-11e5-81d7-79443cc2932d.png)

Added Animations using ngAnimate and ngFx

dmorina commented on bower.json:
```diff
@@ -4,26 +4,27 @@
+    "angular-strap": "2.1.2",
     "angular-ui-sortable": "~0.13.4",
-    "ng-file-upload": "~5.0.9"
+    "bootstrap": "3.3.2",
```
you can drop this

dmorina commented on bower.json:
```diff
@@ -4,26 +4,27 @@
-    "angular-dragdrop": "1.0.9",
+    "angular-route": "1.4.0",
+    "angular-sanitize": "1.4.0",
+    "angular-smart-table": "2.0.2",
```
this one too

dmorina commented on bower.json:
```diff
@@ -4,26 +4,27 @@
+    "angular-route": "1.4.0",
+    "angular-sanitize": "1.4.0",
+    "angular-smart-table": "2.0.2",
+    "angular-strap": "2.1.2",
```
and this

shirishgoyal commented:
Bootstrap packages removed.

dmorina commented:
lgtm, (we will revisit design later)

dmorina commented:
Marking this pr as complete, please don't add other things unless it's a bugfix, thanks

shirishgoyal commented:
@dmorina I am ready with homepage as well. Should I add that to this PR itself or as a new one ?

dmorina commented:
@shirishgoyal would be nice if you submit it as new PR

aginzberg commented:
thanks for pushing this through shirish et al. under my account, it would be nice to have more space on the left before "welcome, .." otherwise lgtm for now and we'll definitely keep iterating :grinning:

shirishgoyal commented:
Alignment fixed and hopefully it looks fine now.

![image](https://cloud.githubusercontent.com/assets/315012/9721971/830da2be-55c2-11e5-874f-0ccada2d23f6.png)

dmorina commented:
fix merge conflicts @shirishgoyal

dmorina commented:
and please for the /task view give 66% to the left view and 33 to the right one, because the left view is the main one where the worker will submit his answers

dmorina commented:
lgtm

nistala commented on bower.json:
```diff
@@ -4,26 +4,24 @@
+    "ng-grid": "2.0.14",
+    "ngFx": "~1.1.0",
+    "underscore": "1.8.3"
   },
```
we haven't removed all references to smart table and angular strap.

nistala commented:
platform depends on packages removed by this PR - angular strap and smart table.
